src/components/create.rs

```diff
@@ -30,14 +42,36 @@ pub(crate) fn Create() -> Element {
     let npub_arbitrator = use_signal(String::new);
     let amount_buyer = use_signal(String::new);
     let amount_seller = use_signal(String::new);
-    let fee_rate = use_signal(|| "1".to_string());
+    let mut fee_rate = use_signal(String::new);
+    let fee_estimates = use_signal(|| Option::<HashMap<u16, f64>>::None);
```

Comment:
You can use this type alias: https://github.com/storopoli/scrow/blob/b21f93dab8523e24f8148d43cd9c48bc016e84e1/src/esplora.rs#L12

```suggestion
    let fee_estimates = use_signal(|| Option::<FeeEstimate>::None);
```

Reply:
Done